test/restart-failed.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var debug = require('debug')('strong-cluster-control:test');
+var cluster = require('cluster');
+var control = require('../');
+
+var SIZE = global.SIZE;
+
+if (cluster.isWorker) {
+  if (cluster.worker.id > SIZE)
+    process.exit(3);
+  return;
+}
+
+var tap = require('tap');
+
+tap.test('good workers are not killed', function(t) {
+  var ended;
+
+  control.start({size: SIZE});
+
+  function size() {
+    return Object.keys(cluster.workers).length;
+  }
+
+  control.once('resize', function() {
+    if (ended) return;
```

Comment:
no, resize event name is probably misleading, it would be better called "reached-configured-size", or something. In other words
1. start
2. resize event occurs once SIZE workers are started
3. call restart

The if ended isn't necessary, I'll remove, its a vestige of when it was `control.on('resize'`.